presto-main/src/main/java/com/facebook/presto/operator/aggregation/TypedSet.java

```diff
@@ -136,7 +139,7 @@ private void addNewElement(int hashPosition, Block block, int position)
     {
         elementType.appendTo(block, position, elementBlock);
         if (elementBlock.getSizeInBytes() > FOUR_MEGABYTES) {
-            throw exceededLocalLimit(new DataSize(4, MEGABYTE));
+            throw new PrestoException(EXCEEDED_FUNCTION_MEMORY_LIMIT, String.format("the argument of %s exceeds of %s", functionName, MAX_FUNCTION_MEMORY));
```

Comment:
For static imports, I don't know of a way to do it automatically. We have Checkstyle for a few things like `toIntExact`, but in general it's a judgement call. We should probably add `String.format` to the blacklist. The rules are in https://github.com/prestodb/presto/blob/master/src/checkstyle/checks.xml